src/components/Articles/Article/dummy.json

```diff
@@ -0,0 +1 @@
imes, serif"},{"offset":20,"length":9,"style":"ITALIC"}],"entityRanges":[],"data":{"text-align":"start"}},{"key":"dlhbs","text":"React apps are static, therefore any update made to the data being displayed will involve a fetch request to a backend service (or incoming data packets from an open socket connection), but in this brief moment the app will be waiting for that data to return to the client. A progress bar is needed — which can easily be linked with your Redux state management to either show or hide the progress bar. In this article we will be visiting how this is implemented in React.","type":"unstyled","depth":0,"inlineStyleRanges":[{"offset":0,"length":471,"style":"color-rgba(0,0,0,0.84)"},{"offset":0,"length":471,"style":"bgcolor-rgb(255,255,255)"},{"offset":0,"length":471,"style":"fontsize-21"},{"offset":0,"length":471,"style":"fontfamily-medium-content-serif-font, Georgia, Cambria, \"Times New Roman\", Times, serif"}],"entityRanges":[],"data":{"text-align":"start"}},{"key":"fc8la","text":"A Little About Progress Bars","type":"header-four","depth":0,"inlineStyleRanges":[{"offset":0,"length":28,"style":"color-rgba(0,0,0,0.84)"},{"offset":0,"length":28,"style":"bgcolor-rgb(255,255,255)"},{"offset":0,"length":28,"style":"fontsize-26"},{"offset":0,"length":28,"style":"fontfamily-medium-content-sans-serif-font, \"Lucida Grande\", \"Lucida Sans Unicode\", \"Lucida Sans\", Geneva, Arial, sans-serif"}],"entityRanges":[],"data":{"text-align":"start"}},{"key":"8o9bd","text":"A progress bar is a singleton and should only be mounted once within your DOM. This makes sense since there is only one App object — e.g. there are not multiple apps running in one browser window. In the event that the browser is split into two or more isolated UIs, there may be a use case for multiple progress bars, although this scenario is probably only likely for complex CMSs — and then you would probably opt for loader animations within that UI space.","type":"unstyled","depth":0,"inlineStyleRanges":[{"offset":0,"length":19,"style":"color-rgba(0,0,0,0.84)"},{"offset":20,"length":9,"style":"color-rgba(0,0,0,0.84)"},{"offset":30,"length":430,"style":"color-rgba(0,0,0,0.84)"},{"offset":0,"length":19,"style":"bgcolor-rgb(255,255,255)"},{"offset":20,"length":9,"style":"bgcolor-rgb(255,255,255)"},{"offset":30,"length":430,"style":"bgcolor-rgb(255,255,255)"},{"offset":0,"length":19,"style":"fontsize-21"},{"offset":20,"length":9,"style":"fontsize-21"},{"offset":30,"length":430,"style":"fontsize-21"},{"offset":0,"length":19,"style":"fontfamily-medium-content-serif-font, Georgia, Cambria, \"Times New Roman\", Times, serif"},{"offset":20,"length":9,"style":"fontfamily-medium-content-serif-font, Georgia, Cambria, \"Times New Roman\", Times, serif"},{"offset":30,"length":430,"style":"fontfamily-medium-content-serif-font, Georgia, Cambria, \"Times New Roman\", Times, serif"},{"offset":20,"length":9,"style":"ITALIC"}],"entityRanges":[],"data":{"text-align":"start"}}],"entityMap":{}}
```

Comment:
what is the purpose of this file?

Reply:
I used this for testing the way draftjs convert raw that it uses. It is no longer needed. I am going to delete it!